### DISCUSSION TITLE
Failed to set up

### DISCUSSION THREAD

**Discussion by Sagi-bb** [humbertogontijo/homeassistant-roborock#60] [Q&A]
It used to work fine but after a few updates it stopped working .
Any ideas ?
![image](https://user-images.githubusercontent.com/68748153/209943874-05912e6a-193f-402e-b13a-6b70258324fb.png)
![image](https://user-images.githubusercontent.com/68748153/209943915-6d4b1d85-e903-44f6-bd89-3eeab7c71b58.png)

Tried to delete and reconfigure without success.

**Comment by Xyaren** [comment]
Duplicate #58 #59
